src/parsing.jl

```diff
@@ -1,5 +1,5 @@
 "A wrapper for one value in a PostgreSQL result."
-struct PQValue{OID}
+struct PQValue{OID, binary_format}
```

Comment:
```suggestion
struct PQValue{OID, BinaryFormat}
```
Type parameters are PascalCase in Julia